allennlp/commands/elmo.py

```diff
@@ -110,6 +111,11 @@ def add_subparser(self, name: str, parser: argparse._SubParsersAction) -> argpar
                 help='The path to the ELMo weight file.')
         subparser.add_argument('--batch-size', type=int, default=DEFAULT_BATCH_SIZE, help='The batch size to use.')
         subparser.add_argument('--cuda-device', type=int, default=-1, help='The cuda_device to run on.')
+        subparser.add_argument(
+                '--use-sentenc-keys',
+                type=bool,
```

Comment:
Maybe this is better as:
```
subparser.add_argument('--use-sentence-keys', 
                       help='Whether to use line numbers or sentence keys as ids.'
                       action='store_true')
```

That way you can just pass it as a flag instead of having to use `--use-sentence-keys True`